sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/work/text/TextNormalisation.scala

```diff
@@ -0,0 +1,9 @@
+package uk.ac.wellcome.models.work.text
+
+object TextNormalisation {
+  def trimTrailing(s: String, c: Char): String =
+    s.replaceAll(s"[\\s]*[$c][\\s]*$$", "")
```

Comment:
I'd like a brief comment to explain this regex, or better no regex at all! The `$$` is especially odd.

Or possibly something like:

```scala
s.trim.stripSuffix(s).trim
```

?

Reply:
My understanding is that `s.trim` removes whitespace at the beginning and end which is not the same.  I don't believe there is an `rtrim`.  I considered breaking it down, but that means multiple regexp calls.

Reply:
Ack, yes. Okay, a comment explaining what the regex is meant to do then.

Reply:
👍